### PR TITLE
coreos-assembler: skip jobs if only docs change

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

File: ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-main-presubmits.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 presubmits:
   coreos/coreos-assembler:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```
Comment (gursewak1997) [inline ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-main-presubmits.yaml:4]: Just to clarify, we do need to set `always_run` to false in order for `skip_if_only_changed` to work?

**Review by gursewak1997 — APPROVED** [review]
/lgtm

File: ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-main-presubmits.yaml (hunk 1 above)
Comment (miabbott) [reply ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-main-presubmits.yaml:4]: Correct; they are exclusive.  (As the CI helpfully told me in a previous revision :laughing: )

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/25933/pull-ci-openshift-release-master-config/1488975580392067072

**Pull request merged**